controllers/backupcronjob/rbac.go

```diff
@@ -0,0 +1,153 @@
+//
+// Copyright (c) 2019-2025 Red Hat, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	dw "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+	"github.com/devfile/devworkspace-operator/pkg/constants"
+	"github.com/devfile/devworkspace-operator/pkg/infrastructure"
+	"github.com/devfile/devworkspace-operator/pkg/provision/sync"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	JobRunnerSAName = "devworkspace-job-runner"
+)
+
+func (r *BackupCronJobReconciler) ensureJobRunnerRBAC(ctx context.Context, workspace *dw.DevWorkspace) error {
+	saName := JobRunnerSAName + "-" + workspace.Status.DevWorkspaceId
+	sa := &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{Name: saName, Namespace: workspace.Namespace, Labels: map[string]string{
+			constants.DevWorkspaceIDLabel: workspace.Status.DevWorkspaceId,
+		}},
+	}
+
+	// Create or update ServiceAccount
+	if err := controllerutil.SetControllerReference(workspace, sa, r.Scheme); err != nil {
+		return err
+	}
+
+	clusterAPI := sync.ClusterAPI{
+		Client: r.Client,
+		Scheme: r.Scheme,
+		Logger: r.Log,
+		Ctx:    ctx,
+	}
+
+	_, err := sync.SyncObjectWithCluster(sa, clusterAPI)
+	if err != nil {
+		if _, ok := err.(*sync.NotInSyncError); !ok {
```

Comment:
@dkwon17 
Do we really need to ignore NotInSyncError? 
Should we try on the next reconcile loop?